data_structures/red_black_tree.ts

```diff
@@ -109,7 +109,7 @@ export class RedBlackTree<T> extends BinarySearchTree<T> {
   constructor(compare: (a: T, b: T) => number = ascend) {
     if (typeof compare !== "function") {
       throw new TypeError(
-        "compare must be a function, did you mean to call RedBlackTree.from?",
+        "The 'compare' parameter must be a function, did you mean to call RedBlackTree.from?",
```

Comment:
```suggestion
        "Cannot construct a RedBlackTree: The 'compare' parameter must be a function, did you mean to call RedBlackTree.from?",
```

Reply:
I think this should also be formulated as the current state: "the compare parameter is not a function"

Reply:
```suggestion
        "Cannot construct a RedBlackTree: The 'compare' parameter is not a function, did you mean to call RedBlackTree.from?,
```